packages/components-native/src/Content/Content.tsx

```diff
@@ -9,10 +9,14 @@ import { useSpaceAroundStyles } from "./ContentSpaceAround.style";
 export type Spacing =
   | "none"
```

Comment:
I think ideally we would just pull these available types from the design package? But just looking for parity here, was not aiming for a refactor.